configuration/scripts/options/set_nml.box2001

```diff
@@ -22,7 +25,10 @@ ktransport     = -1
 coriolis       = 'constant'
 atmbndy        = 'constant'
 atm_data_type  = 'box2001'
+ocn_data_type  = 'box2001'
 ice_data_type  = 'box2001'
+ice_data_conc  = 'p5'
```

Comment:
I think the setup is fine, but having this ice_data_conc in the set_nml file is a little confusing because it should be overwritten.  If there is a default value = p5, maybe this could simply be left out of the set_nml list.  A better solution is to fix the logic, as noted below.